src/nni_manager/package.json

```diff
@@ -13,6 +13,7 @@
     "azure-storage": "^2.10.2",
     "chai-as-promised": "^7.1.1",
     "child-process-promise": "^2.2.1",
+    "deepmerge": "^4.2.2",
```

Comment:
there should be some updates in yarn.lock needs to be checked in.

Reply:
fixed